frank-flow/src/frontend/src/app/shared/components/edit-dialog/edit-dialog.component.html

```diff
@@ -0,0 +1,27 @@
+<ngx-smart-modal
+  #editDialog
+  (onDataAdded)="onDataAdded()"
+  (onOpenFinished)="inputElement.focus()"
+  (onAnyCloseEvent)="edit()"
+  class="editDialog"
+  customClass="nsm-centered"
+  identifier="editDialog"
+>
+  <div>
+    <h2>
+      Modify {{ currentDirectory?.path ? 'folder' : 'file' }} {{ oldFileName }}
+    </h2>
+  </div>
+  <label for="inputElement">Name</label>
+  <input
+    #inputElement
+    [(ngModel)]="fileName"
+    type="text"
+    (keyup.enter)="editDialog.close()"
+  />
```

Comment:
Place in one row (with input-row).
Add `id` attribute to the input filed and remove the element reference.